plugins/services/src/js/validators/PlacementsValidators.js

```diff
@@ -0,0 +1,130 @@
+import ValidatorUtil from "#SRC/js/utils/ValidatorUtil";
+import { findNestedPropertyInObject } from "#SRC/js/utils/Util";
+import PlacementConstraintsUtil from "../utils/PlacementConstraintsUtil";
+import { PROP_MISSING_ONE, SYNTAX_ERROR } from "../constants/ServiceErrorTypes";
+
+function checkDuplicateOperatorField(constraints) {
+  if (!constraints || ValidatorUtil.isEmpty(constraints)) {
```

Comment:
No need for `!constraints` because `ValidatorUtil.isEmpty` already checks for `null` and undefined.